dojo/models.py

```diff
@@ -998,6 +1038,37 @@ class Product(models.Model):
         blank=False,
         verbose_name=_("Disable SLA breach notifications"),
         help_text=_("Disable SLA breach notifications if configured in the global settings"))
+    async_updating = models.BooleanField(default=False,
+                                            help_text=_('Findings under this SLA configuration are asynchronously being updated'))
```

Comment:
Should this be "Product or SLA configuration"? Seems like this is doing double duty for 1) SLA config itself being modified or 2) different SLA config being chosen for this Product?

```suggestion
                                            help_text=_('Findings under this Product or SLA configuration are asynchronously being updated'))
```